PetScanList/one_page_bot.py

```
@@ -0,0 +1,51 @@
+        return "no changes"
+    # ---
+    summary = "بوت: تحديث قائمة (تجريبي)"
+    # ---
+    save = page.save(newtext, summary=summary)
+    # ---
+    if isinstance(save, dict):
+        # OrderedDict([('result', 'Success'), ('pageid', 9857018), ('title', 'ويكيبيديا:برنامج قراءة ويكيبيديا في الصف - اليمن 2/مقالات مقترحة/لا مصدر'), ('contentmodel', 'wikitext'), ('oldrevid', 69427827), ('newrevid', 69427829), ('newtimestamp', '2025-01-29T00:41:53Z')])
+        if save.get("result") == "Success":
+            te = "save success"
+        else:
+            te = str(save)
```

Ruff (0.8.2)

Use ternary operator `te = "save success" if save.get("result") == "Success" else str(save)` instead of `if`-`else`-block

Replace `if`-`else`-block with `te = "save success" if save.get("result") == "Success" else str(save)`

(SIM108)